static/client.js

```diff
@@ -1045,7 +1060,7 @@ function init() {
         return false;
     });
     if(Math.random() < .5)
-        $("body").css("background-image", "url(http://i.imgur.com/9Pia8.jpg)");
+        $("body").css("background-image", "url(https://i.imgur.com/9Pia8.jpg)");
```

Comment:
Thanks. If you want to help remove this external dependency, I've already uploaded this image to the assets folder.